optuna/integration/chainer.py

```diff
@@ -28,15 +28,12 @@ class ChainerPruningExtension(Extension):
 
     Example:
 
-        Add a pruning extension which observes validation losses to
+        See `the example <https://github.com/optuna/optuna/blob/master/
+        examples/pruning/chainer_integration.py>`__
+        if you want to add a pruning extension which observes validation accuracy to
         `Chainer Trainer <https://docs.chainer.org/en/stable/reference/generated/
         chainer.training.Trainer.html>`_.
```

Comment:
The grammar can be improved a bit.
```
See the example if you want to add a pruning extension which the observes validation accuracy of a Chainer Trainer.
```
Not a strong opinion but I also believe simplifying it as follows is fine too. You can then be consistent across all integrations.
```
See example.
```
where `example` is a hyperlink.

Reply:
Thanks! I fixed it.

> Not a strong opinion but I also believe simplifying it as follows is fine too

It seems that same discussion was held in https://github.com/optuna/optuna/pull/1019#discussion_r394304394 .
I think it is good for users to show what the example actually does.

Reply:
@nuka137, thanks for your fixed. What do you think of my latter suggestion above? Just writing "See example". I think it' less to maintain and there are some grammatical changes I'd like to make otherwise.

Reply:
I just saw your reply. Okay, let's keep it as is. Let me leave some comments on the grammar.